### PR TITLE
CEDS-2935 Connect SFUS to the accessibility statement Frontend service 

### PR DESCRIPTION
**Pull request by PauloPortugal**
This includes removing the current accessibility page and content and
rely completely on a 3rd party service:

The PR can be found here:
   https://github.com/hmrc/accessibility-statement-frontend/pull/321/files

### CODE REVIEW

File: app/views/accessibility_statement.scala.html [hunk 1]
```diff
@@ -14,71 +14,101 @@
...
+      messages("fuf.accessibility.statement.using.l3"),
+      messages("fuf.accessibility.statement.using.l4"),
+      messages("fuf.accessibility.statement.using.l5")
+    )
+   )
+  @paragraphBody(message = messages("fuf.accessibility.statement.using.p3"))
+  @paragraphBody(message = messages("fuf.accessibility.statement.using.p4"))
+
+
+  <h2 id="information-you-need" class="govuk-heading-l">@messages("fuf.accessibility.statement.accessible.heading")</h2>
```
Comment (MaciejRewera) [inline app/views/accessibility_statement.scala.html:59]: The id is repeated (!)

File: conf/application.conf [hunk 2]
```diff
@@ -194,6 +194,7 @@ feedback {
 }
 
 accessibility-link-url = "/cds-file-upload-service/help/accessibility"
+accessibility-statement.service-path = "/cds-file-upload-service/help/accessibility"
```
Comment (MaciejRewera) [inline conf/application.conf:197]: IMHO it would be better to keep urls in `urls` section

**Review by MaciejRewera — COMMENTED** [review]
Left a few minor comments.

File: app/views/accessibility_statement.scala.html (hunk 1 above)
Comment (PauloPortugal) [reply app/views/accessibility_statement.scala.html]: Does no longer apply to the current state of the PR

File: conf/application.conf (hunk 2 above)
Comment (PauloPortugal) [reply conf/application.conf]: No it has to be there.

File: conf/application.conf [hunk 3]
```diff
@@ -193,7 +193,7 @@ feedback {
   url = "http://localhost:9514/feedback/FUS"
 }
 
-accessibility-link-url = "/cds-file-upload-service/help/accessibility"
+accessibility-statement.service-path = "/cds-file-upload"
```
Comment (MaciejRewera) [inline conf/application.conf:196]: I am still not convinced about this place in the config. But I guess it is already being used in different environment configurations.

Comment (PauloPortugal) [reply conf/application.conf:196]: This is how we need to add the config. The Accessibility Statement service expects that config there.
And it does not work seamlessly in local dev as well, as is normally the case with these `play-ui` integrations.
I would have happily put it on another structure or make it running locally, but if we want to follow the service integration this is it. we cannot change.

Comment (PauloPortugal) [reply conf/application.conf:196]: Even trying to overrride that service-path with a protocol, domain and port, does not work, as they will just append that at the end: 
This is how we check our statement locally: https://github.com/hmrc/cds-file-upload-frontend/pull/95/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R61

**Review by MaciejRewera — APPROVED** [review]

**Pull request merged**